bower.json

```diff
@@ -11,11 +11,11 @@
     "app-router": "^2.6.1",
     "firebase-element": "GoogleWebComponents/firebase-element#^1.0.14",
     "polymerfire": "firebase/polymerfire#^0.9.2",
-    "Chart.js": "^2.1.6",
+    "Chart.js": "2.1.6",
     "chart-elements": "^3.0.2",
     "iron-media-query": "PolymerElements/iron-media-query#^1.0.8",
     "progress-bubble": "^1.3.0",
-    "paper-range-slider": "IftachSadeh/paper-range-slider#^0.1.2",
+    "paper-range-slider": "IftachSadeh/paper-range-slider#^0.2.4",
```

Comment:
I would remove `^` sign from version for that library as its versioning is unpredictible